pkgs/by-name/dp/dpp/package.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libopus
+, openssl
+, zlib
+, libsodium
+, gcc
+, pkg-config
+, autoPatchelfHook
+, lib
+}:
+stdenv.mkDerivation rec {
+  pname = "dpp";
+  version = "10.0.29";
+
+  src = fetchFromGitHub {
+    owner = "brainboxdotcc";
+    repo = "DPP";
+    rev = "v${version}";
+    hash = "sha256-BJMg3MLSfb9x/2lPHITeI3SWwW1OZVUUMVltTWUcw9I=";
+  };
+
+  nativeBuildInputs =  [
+    cmake
+    libopus
+    pkg-config
+  ] ++ lib.optional (stdenv.hostPlatform.isLinux) autoPatchelfHook;
+
+  buildInputs = [
+    openssl
+    zlib
+    libsodium
+    libopus
+    pkg-config
+  ];
+
+  cmakeFlags = let
+      libExt = stdenv.hostPlatform.extensions.sharedLibrary;
+   in [
+    "-DOPENSSL_CRYPTO_LIBRARY=${openssl.out}/lib/libcrypto${libExt}"
+    "-DOPENSSL_SSL_LIBRARY=${openssl.out}/lib/libssl${libExt}"
+  ];
+
+  meta = {
+    description = "Discord C++ Library";
+    homepage = "https://github.com/brainboxdotcc/DPP";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ rgchavezchicoma ];
```

Comment:
You have to add yourself to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix in a separate commit.

Reply:
done